TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/AndroidMovementService.java

```diff
@@ -15,7 +15,7 @@
 import com.novoda.notils.logger.toast.Toaster;
 import com.novoda.support.Optional;
 
-public class AndroidMovementService extends Service implements MovementService {
+public class AndroidMovementService extends Service implements UsbChangesListener {
```

Comment:
In a follow-up PR we should hide the usage of `Service`. There should be `MovementService` (not an android service` which occurs in an android service but is not dependent on it. Ideally the `Service` class will have very little code just like the activities.